assets/js/blocks/single-product-details/style.scss

```diff
@@ -0,0 +1,46 @@
+.wp-block-woocommerce-single-product-details {
```

Comment:
If we resolve #8314, is this CSS still necessary?

Reply:
I hope we don't need it anymore after solving that issue; however, if we need to change some things in the UX I think we still are going to need this